frontend/src/screens/apps/Create.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+
+const Create = () => {
```

Comment:
Can we rename this file and component?

This is what you see post-creation of an app, right?

Reply:
Renamed to `AppCreated`